### PR TITLE
feat(player): Show the current chapter in player controls ui

### PR DESCRIPTION
**Pull request by abdallahmehiz**
Also "optimize?" the logic for highlighting the correct chapter in the the chapters sheet
## Here is a preview of how it looks
Screenshot_20240127-125826

### CODE REVIEW

**Comment by abdallahmehiz** [comment]
was planning to make it draggable so swiping up and down would seek between chapters but i don't think i am gonna do that anymore

File: app/src/main/java/eu/kanade/tachiyomi/ui/player/viewer/components/CurrentChapter.kt [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+    private fun CurrentChapterComposable(
+        chapter: Chapter,
+        modifier: Modifier = Modifier,
+    ) {
+        Box(
+            modifier = modifier
+                .clip(RoundedCornerShape(25))
+                .background(MaterialTheme.colorScheme.background.copy(alpha = 0.6F))
+                .padding(horizontal = MaterialTheme.padding.large)
+                .wrapContentSize(Alignment.CenterStart)
```
Comment (Samfun75) [inline app/src/main/java/eu/kanade/tachiyomi/ui/player/viewer/components/CurrentChapter.kt:80]: lintError?
```suggestion
                .wrapContentSize(Alignment.CenterStart),
```

Comment (abdallahmehiz) [reply app/src/main/java/eu/kanade/tachiyomi/ui/player/viewer/components/CurrentChapter.kt]: didn't notice lol

File: app/src/main/java/eu/kanade/tachiyomi/ui/player/viewer/components/CurrentChapter.kt [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+                        overflow = TextOverflow.Clip,
+                        color = MaterialTheme.colorScheme.tertiary,
+                    )
+                    currentChapter.title?.let {
+                        Text(
+                            text = " • ",
+                            textAlign = TextAlign.Center,
+                            maxLines = 1,
+                            overflow = TextOverflow.Clip,
+                        )
```
Comment (abdallahmehiz) [inline app/src/main/java/eu/kanade/tachiyomi/ui/player/viewer/components/CurrentChapter.kt:118]: why not use `DotSeperatorText()` instead?
```suggestion
                        DotSeperatorText()
```

Comment (jmir1) [reply app/src/main/java/eu/kanade/tachiyomi/ui/player/viewer/components/CurrentChapter.kt:118]: cause in vertical mode, i want this to be cut off if there isn't enough space

**Review by jmir1 — APPROVED** [review]

**Pull request merged**